jarvis_files/launch_scripts/loc

```diff
@@ -6,27 +6,26 @@ xdotool windowfocus $WID
 
 # Enter commands you want to be executed in the window here
 
-# Run Localization
+# Check If need to ssh first
+if [[ $1 ]]; then
+    ./jarvis_files/launchScripts/ssh $WID
+fi
 xdotool key type "sudo ./jarvis exec jetson/gps"
 xdotool key KP_Enter
 sleep 0.4s
 xdotool key type "mrover"
 xdotool key KP_Enter
 
-#Open a new tab for Obstacle LCM
+#Open a new tab
```

Comment:
Just small thing: but in the comments let's keep the spacing consistent, and at least add what tab is being run